cfme/tests/openstack/infrastructure/test_roles.py

```diff
@@ -21,8 +21,8 @@
 
 @pytest.yield_fixture(scope="module")
 def roles(appliance, provider):
-    collection = DeploymentRoleCollection(appliance=appliance).filter({'provider': provider})
-    roles = collection.all(provider)
+    collection = DeploymentRoleCollection(appliance).filter({'provider': provider})
```

Comment:
REQUIRED: Now you should use `collection = appliance.collections.deployment_roles.filter({'provider': provider})`

Reply:
Right, until the provider supports collections, and then it will be
```provider.collections.deployment_roles```
without the need for an explicit filtering

Reply:
`AttributeError: Collection [deployment_roles] not known to applinace`.. May it have other name? 
I've tried `deployment_role` - the same result